### PR TITLE
Update codeOwner

### PR DESCRIPTION
**Pull request by aidan-kwon**
## Proposed changes

Update `.github/CODEOWNERS`
CodeOwners, please check this change and leave your comments. 

CodeOwners 
- are responsible for the change of their directory. 
- are supposed to review PRs 
- should develop their capability to maintain their code correct and clean 

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by jeongkyun-oh — APPROVED** [review]

**Review by ehnuje — DISMISSED** [review]

**Review by hqjang-pepper — DISMISSED** [review]

**Review by mckim19 — APPROVED** [review]

**Review by KimKyungup — DISMISSED** [review]

**Review by yoomee1313 — DISMISSED** [review]

**Review by kjhman21 — APPROVED** [review]

**Review by aeharvlee — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -36,31 +36,31 @@
...
-/accounts/      @KimKyungup @aidan-kwon @jeongkyun-oh @yoomee1313 @hqjang-pepper
-/api/           @KimKyungup @aidan-kwon @jeongkyun-oh @yoomee1313 @jimni1222 @sirano11 @aeharvlee @mckim19
-/blockchain/    @KimKyungup @aidan-kwon @jeongkyun-oh @ehnuje @winnie-byun @jimni1222 @sirano11 @mckim19
-/build/         @KimKyungup @aidan-kwon @jeongkyun-oh
-/client/        @aidan-kwon @jeongkyun-oh @hqjang-pepper
-/cmd/           @KimKyungup @aidan-kwon @jeongkyun-oh @yoomee1313
+/accounts/
+/api/           @jimni1222 @sirano11 @aeharvlee
+/blockchain/    @KimKyungup @aidan-kwon @jeongkyun-oh @ehnuje
+/build/         @henry-will @hyunsooda @hqjang-pepper @nohkwak @2dvorak
```
Comment (henry-will) [inline .github/CODEOWNERS:42]: I don't know about this file and change history.
I think build directory's owner should include @KimKyungup @aidan-kwon @jeongkyun-oh.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -36,31 +36,31 @@
...
+/client/        @aidan-kwon @aeharvlee @hyunsooda
+/cmd/           @aidan-kwon @henry-will @hyunsooda @hqjang-pepper @nohkwak
```
Comment (henry-will) [inline .github/CODEOWNERS:44]: If you want to include servicechain part, please add @2dvorak

**Review by henry-will — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (aidan-kwon) [reply .github/CODEOWNERS]: I re-arranged service chain developers. Please, take a look again.

**Review by kjhman21 — APPROVED** [review]

**Review by jeongkyun-oh — APPROVED** [review]

**Review by hqjang-pepper — APPROVED** [review]

**Comment by aidan-kwon** [comment]
@henry-will Could you check this again?

**Review by henry-will — APPROVED** [review]
Looks good.

**Pull request merged**